bundles.config.js

```diff
@@ -474,6 +480,10 @@ exports.extensionBundles.forEach(c => {
   console./*OK*/assert('name' in c, `name key must exist. Found ${strRep}.`);
   console./*OK*/assert('version' in c, 'version key must exist. ' +
       `Found ${strRep}.`);
+  if (Array.isArray(c.version)) {
```

Comment:
there are two different entries for `amp-story`. One for `0.1` and one for `1.0`. I am not sure how this handles that case. I feel like the build order wins there and `latest` ends up `1.0` so it is not locked.

Reply:
In that case, we'll probably have to enforce the presence of `latestVersion` and make sure that two extensions with the same name cannot have different values for it. WDYT? @jridgewell

Reply:
Done. Enforced the presence of `latestVersion` and added a check for extensions with the same name.